site/index.html

```diff
@@ -76,6 +76,52 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Rapid prototyping</h3>
+            <p>Start small and explore new ideas to find your market fit. Haskell is a compact language with a large ecosystem of open-source libraries and tools
+               to get you started and help you growing.
+               The strong type system makes it easy to write programs that work, both large and small, even when you don't like writing types &mdash; the compiler will figure it out.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Scale with ease</h3>
+            <p>From 1 to 100k users, and more. Haskell offers one of the best support for async, concurrent and parallel programming.
+               It can take advantage of multi-core processors with ease and confidence like few other languages can
+               &mdash; while keeping complexity low as your user base and code base grows.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Maintain with confidence</h3>
+            <p>Add complex features or refactor large parts of your code base reliably. Haskell has one of the strongest type systems on the market to prevent mistakes
+               before they reach production, saving on total software lifetime costs.&mdash;“If it compiles, it works” and "Fearless refactoring" are common proverbs among Haskellers.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Bleeding edge</h3>
+            <p>Get a view into the latest programming innovations. Haskell has its roots in programming language research, has already influenced many popular languages and continues to do so.
+               Experiment with new programming techniques like Effect Systems, Linear Types and more related to Dependent Types for even better reliability when you can tolerate some immaturity;
+               or stay with stable best practices&mdash;Haskell supports both.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Delightful</h3>
+            <p>Software that works, scales and is easy to maintain helps you to attract talent and keep your team motivated.
+               Haskell is a language that lets you express your ideas clearly and teaches you a new way of thinking about programming
+               &mdash; if you work on a challenging project, you might as well have a good time.
```

Comment:
I rewrote this section and tried to highlight some actual properties. I think it's worth noting that Haskell **is a new programming paradigm**. We should say so. And we should also give some specific details whenever we make claims.

```suggestion
            <h3>A new paradigm</h3>
            <p>Haskell is a language that lets you express your ideas clearly and teaches you a new way of thinking about programming. Based on lambda calculus it features referential transparency, immutability and lazy evaluation. Concepts that will blow your mind.
               &mdash; relearn programming while having an absolute blast.
```

Reply:
We're moving more towards features again, which i would keep to the other section further below on the page, and keep this compact on the value propositions (the Scale subsection being at the border of mentioning too much features already). Here, i like the Fun aspect more as this was highlighted by a lot of Haskellers, because new is not automatically better.

Reply:
I strongly disagree... this section is what connects the experience with the features. Otherwise it's just empty platitudes. This allows the reader to connect the dots.